packages/arb-util/hashing/hashing.go

```diff
@@ -66,6 +74,14 @@ func Uint128(input *big.Int) []byte {
 	return solsha3.Uint128(new(big.Int).Set(input))
 }
 
+func Uint256Array(input []*big.Int) []byte {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function Uint256Array should have comment or be unexported](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)